src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -941,6 +930,19 @@
       <Link>Interop\Windows\BCrypt\Interop.BCryptGenRandom.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="('$(TargetsWindows)'=='true')  And '$(EnableDummyGlobalizationImplementation)' != 'true'">
+    <Compile Condition="'$(EnableWinRT)' != 'true'" Include="System\Globalization\HijriCalendar.Win32.cs" />
```

Comment:
This should stay in its original place - there are dummy version of these (HijriCalendar.Dummy.cs, JapaneseCalendar.Dummy.cs).